dashboard/app/views/plc/enrollment_evaluations/preview_assignments.html.haml

```diff
@@ -0,0 +1,15 @@
+- course_unit = @enrollment_unit_assignment.plc_course_unit
+
+You will be assigned these modules
+- @enrolled_modules.each do |enrolled_module|
+  %li
+    = enrolled_module.name
+
+%br
+You can change your course enrollment here
+= form_tag(action: 'confirm_assignments') do
+  - options = course_unit.plc_learning_modules.pluck(:name, :required, :id).map {|name, required, id| ["#{name}#{' - Required' if required}", id]}
```

Comment:
This logic probably shouldn't live in the view.  Don't we already have a helper for adding `' - Required'` where needed?